napari/_qt/_qapp_model/_tests/test_dock_widget_toggler.py

```diff
@@ -0,0 +1,25 @@
+from napari._app_model._app import get_app
+from napari._qt._qapp_model.qactions._window import ToggleDockWidgetAction
+from napari._qt.qt_main_window import Window
+
+
+def test_dock_widget_toggler(make_napari_viewer):
+    """Tests `ToggleDockWidgetAction` toggling works."""
+    viewer = make_napari_viewer(show=True)
```

Comment:
If test still fails on CI I would try adopt the `show` "strategy" from [the activity dialog tests](https://github.com/napari/napari/blob/b44cf8e880c76adfb7a8f8cd1060d11bcaf04a43/napari/_qt/dialogs/_tests/test_activity_dialog.py#L13)